examples/example-app.yaml

```diff
@@ -52,3 +53,7 @@ spec:
         - "/main"
         - "--process-metrics"
         - "--go-metrics"
+        resources:
+          requests:
+            cpu: 100m
+            memory: 100Mi
```

Comment:
Same here. We give our [operator `32M`](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/main/manifests/operator.yaml#L267C15-L267C15). Did you check actual usage?

Reply:
No, will check, but I don't think that's important as long as it's not 2GBi (default on my autopilot cluster)